addons/test_mail/tests/test_message_post.py

```diff
@@ -281,27 +281,50 @@ def test_post_answer(self):
         self.assertEqual(parent_msg.partner_ids, self.env['res.partner'])
         self.assertEmails(self.user_employee.partner_id, [])
 
+        # post a first reply
+        self._init_mock_build_email()
         msg = self.test_record.with_user(self.user_employee).message_post(
             body='<p>Test Answer</p>',
-            message_type='comment', subtype='mt_comment',
+            message_type='comment',
+            subject='Welcome',
+            subtype='mt_comment',
+            parent_id=parent_msg.id,
             partner_ids=[self.partner_1.id],
-            parent_id=parent_msg.id)
+        )
 
         self.assertEqual(msg.parent_id.id, parent_msg.id)
         self.assertEqual(msg.partner_ids, self.partner_1)
         self.assertEqual(parent_msg.partner_ids, self.env['res.partner'])
 
-        # check notification emails: references
-        self.assertEmails(self.user_employee.partner_id, [[self.partner_1]], ref_content='openerp-%d-mail.test.simple' % self.test_record.id)
-        # self.assertTrue(all('openerp-%d-mail.test.simple' % self.test_record.id in m['references'] for m in self._mails))
+        # check notification emails, among other references
+        self.assertEmails(
+            self.user_employee.partner_id,
+            [[self.partner_1]],
+            subject='Welcome',
+            ref_content='openerp-%d-mail.test.simple' % self.test_record.id,
+            references='%s %s' % (parent_msg.message_id, msg.message_id),
```

Comment:
can you add a comment that references should be sorted from the oldest to the newest ?